test/components/hover-cards.spec.js

```diff
@@ -159,12 +159,30 @@ describe('HoverCards', () => {
       ))
       .request(hover(clock))
       .respondWithData(() => testData.extendedDatasets.last())
-      .respondWithProblem(() => entity)
+      .respondWithProblem()
       .afterResponses(async (component) => {
         should.not.exist(document.querySelector('.popover'));
         component.should.not.alert();
         // Make sure that this does not result in a Vue warning.
         await getAnchor(component).trigger('mouseleave');
       });
   });
+
+  it('does not log an error if mouseleave is triggered during request', () => {
```

Comment:
In case you're wondering how this test works (there's not much in the way of an assertion), our tests automatically fail if Vue logs a warning. Before the fix, Vue was logging a warning, but with the fix in place, it no longer logs a warning.